packages/databases-collections/src/components/rename-collection-modal/rename-collection-modal.tsx

```diff
@@ -100,7 +100,7 @@ function RenameCollectionModal({
   const onFormSubmit = () => {
     if (modalState === 'confirmation-screen') {
       setModalState('input-form');
-      renameCollection(newName);
+      renameCollection(newName.trim());
```

Comment:
Ditto about this being business logic that belong in the action, not view. Would be nice to also not to batch dispatch actions as suggested by the redux style guide, but as this is existing code, I'll leave it to you whether or not you want to clean it up or not